v23.10.3-postgres/docker-compose.yml

```
@@ -0,0 +1,53 @@
+version: "3.5"
+services:
+  db:
+    build:
+      context: .
+      dockerfile: postgres.dockerfile
+    container_name: postgres-network
+    restart: always
+    shm_size: 128mb
+    environment:
+      - POSTGRES_PASSWORD=docker
+    ports:
+      - "5432:5432"
+    healthcheck:
+      test: "pg_isready --username=docker && psql --username=docker --list"
+      timeout: 5s
+      retries: 20
+    networks:
+      - internal-net
+
+  planet9:
+    build:
+      context: .
+      dockerfile: p9.dockerfile
+    container_name: planet9-network
+    depends_on:
+      - db
+    environment:
+      - P9_SERVER_SCRIPTS_NPM=/home/planet9/npm
```

```suggestion
      - P9_SERVER_SCRIPTS_NPM=/home/planet9/npm/serverscript
      - P9_WEBAPP_DIR=/home/planet9/npm/webapp
```
Should we add both directories in the persistent volume?

---

Definitely!